README.md

```diff
@@ -21,14 +21,32 @@ If some mutants are not killed, it means that the specification has issues and i
 
 [**`move-mutator`**](move-mutator/README.md) is a helper tool that mutates Move source code. Every modification / mutation is called a mutant. _Note that the tool mutates only the source code; tests and spec blocks are unaffected and are not mutated by this tool._
 
-## Quick introduction to mutation tools
+## Installation
+
+### Via Aptos CLI (Recommended)
 
-This is a quick tutorial on how the `move-mutation-test` tool should be used. Install the tool with:
+The easiest way to install and update `move-mutation-test` is through the Aptos CLI:
+
+```bash
+# Install the latest version
+aptos update move-mutation-test
 
+# Install/Update a specific version (e.g., v2.0.0)
+aptos update move-mutation-test --target-version v2.0.0
 ```
-$ RUSTFLAGS="--cfg tokio_unstable" cargo install --git https://github.com/eigerco/move-mutation-tools.git --locked move-mutation-test
```

Comment:
@Rqnsom I removed this, but **I'm not sure** if it will work without it.
It's actually not removed, but added in the CI, because it doesn't take in consideration the `.cargo/config.toml`.

Can you please try to test it locally, because locally for me it compiles fine?

Reply:
For me, the local run on my Linux machine requires the `RUSTFLAGS="--cfg tokio_unstable"` flag.

Otherwise I get this compilation error:
```
error[E0599]: no method named `disable_lifo_slot` found for mutable reference `&mut tokio::runtime::Builder` in the current scope
  --> /home/user/.cargo/git/checkouts/aptos-core-ed28e035b32a7f02/8a0f983/crates/aptos-runtimes/src/lib.rs:47:10
   |
41 | /     builder
42 | |         .thread_name_fn(move || {
43 | |             let id = atomic_id.fetch_add(1, Ordering::SeqCst);
44 | |             format!("{}-{}", thread_name_clone, id)
45 | |         })
46 | |         .on_thread_start(on_thread_start)
47 | |         .disable_lifo_slot()
   | |         -^^^^^^^^^^^^^^^^^ private field, not a method
   | |_________|
```